packages/snfoundry/contracts/src/Staker.cairo

```diff
@@ -45,7 +46,7 @@ pub mod Staker {
     #[storage]
     struct Storage {
         eth_token_dispatcher: IERC20CamelDispatcher,
-        balances: LegacyMap<ContractAddress, u256>,
+        balances: Map<ContractAddress, u256>,
```

Comment:
the [readme ](https://github.com/Scaffold-Stark/speedrunstark/tree/nadai/challenge-1-update?tab=readme-ov-file#checkpoint-1--staking-) need to update of using Map as well